docs/Developers/Git_Basics.rst

```diff
@@ -542,3 +542,67 @@ also reorder the commit listing to change the order of the commits themselves.
 If you would like more information/commands, please use your favourite search
 engine to look for it. Git is widely used throughout the world and there are
 many good tutorials and git related Q&A threads out there.
+
+
+::
+
+    $ git reflog
+
+Will present a list of all actions performed on git and across all branches.
+On running ``git reflog`` each command performed till now will be presented
+on the terminal and each command will have an index. The structure of the
+index is HEAD@{index}.This command is very important when you are in a mess.
+The next command will provide the users a weapon to fix their mistakes.
+
+
+::
+
+   $ git reset HEAD@{index}
+
+Reset to the point before you started messing up. HEAD@{index} denotes the
```

Comment:
Enclose " HEAD@{index} " with two backquotes ( ``  ) like you have done at other places